### DISCUSSION TITLE
Extending current basemaptiler profile with other osm tags

### DISCUSSION THREAD

**Discussion by Lychfindel** [onthegomap/planetiler#290] [Q&A] [answered]
Hi,
I'm trying to generate a tiles set that uses the standard OpenMapTiles schema, but adding some tags in the POI, for example the id.
Is there an easy way to do it without writing a complete new profile?

**Comment by ZeLonewolf** [comment]
The long term vision is to continue expanding the functionality in #160 to the point where openmaptiles can be completely generated by configuration.  Then, it would just be a matter of editing the config file.  Unfortunately that's quite a ways off.

**Reply by Lychfindel** [reply]
It will be very interesting when it will be fully configurable! In the meantime the solution from @wipfli works great!

**Comment by wipfli** [comment, marked as answer]
I once hacked something together to modify OMT. See https://gist.github.com/wipfli/02ba9081b26c3a0cfae054e5c5402fe7

**Reply by Lychfindel** [reply]
Thank you! It works perfectly!
